lib/ui/prompt.js

```diff
@@ -72,8 +72,9 @@ PromptUI.prototype.processQuestion = function (question) {
 };
 
 PromptUI.prototype.fetchAnswer = function (question) {
+  var screen = this.createScreen();
```

Comment:
We'll be able to get rid of this function once we move the initialization to the constructor.